### ISSUE TITLE
Edits in Swift Photo App 

### ISSUE THREAD

**Issue by awwalker** [minio/docs.minio.io#113]
1. In the first paragraph of text there is an extra space between the end of here and the colon in "Full code is available here :"

2. Incorrectly formatted comments in ViewController.swift --
 
- none of the comments in the first "@IABaction func" are capitalized or ended with a period.
- the last two comments are also missing capitalization and periods

3. There are exclamations after in task!.resume() and in NSURLSession.sharedSession().dataTaskWithURL(url) { (data, response, error) in
            completion(data: data, response: response, error: error)
            }!.resume()
(Using XCode 7.3.1 the IDE complains)

4. In the "@IAB func refreshButton" the let url line should be         

let url = NSURL(string: "http://play.minio.io:8080/MinioJavaRESTExample-0.0.1-SNAPSHOT/minio/photoservice/list")

5. Incorrect numbering there are two number 3's (MainStoryBoard and ViewController.swift) 

6. Missing period after the line of text in Number 4 Info.plist

7. Inconsistent periods in the bulletin points in Number 3 MainStoryBoard only the last bullet point has a period the others don't.

**Comment by deekoder** [comment]
Harsha can you update the war file to be called PhotoServiceAPI instead of MinioJavaRESTExample-0.0.1

**Comment by deekoder** [comment]
1. Done
2. Done
3. Xcode 7.3 was since released. The ide itself enables user to correct this in the latest Xcode. So i think we need not do anything about this. Removing exclamation mark will however mess up Xcode 7.2
4. Blocked on this
5. Done
6. Done
7. Done

**Comment by deekoder** [comment]
Opened a new PR for updating code with PhotoAPIService in the URL
